manifest.yaml

```diff
@@ -1,7 +1,8 @@
+models:
+    - specs/models/unet2d/nuclei_broad/UNet2DNucleiBroad.model.yaml
 transformations:
-    - specs/transformations/BCELoss.transformation.yaml
+    - specs/losses/BCELoss.transformation.yaml
```

Comment:
I thought we agreed that there is no difference between transformatins and losses, at least on the spec level?

Reply:
true, but also specs may lie anywhere. For more organization I split the transformations specs into the two folders transformations and losses...

Reply:
Hmm, I still find this potentially confusing, if we don't have the concept of `loss` in the spec, we shouldn't put it into another folder here.
Sure, it's allowed, but doesn't mean it's good practice.